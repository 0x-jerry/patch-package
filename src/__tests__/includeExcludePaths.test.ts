import { runShellTest } from "./shellTests"

runShellTest("include-exclude-paths")
