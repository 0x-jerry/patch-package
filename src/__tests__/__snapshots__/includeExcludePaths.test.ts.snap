// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test include-exclude-paths: exclude all but flip 1`] = `
"SNAPSHOT: exclude all but flip
patch-package
--- a/node_modules/lodash/flip.js
+++ b/node_modules/lodash/flip.js
@@ -25,4 +25,4 @@ function flip(func) {
   return createWrap(func, WRAP_FLIP_FLAG);
 }
 
-module.exports = flip;
+module.patchPackage = flip;
END SNAPSHOT"
`;

exports[`Test include-exclude-paths: no base files 1`] = `
"SNAPSHOT: no base files
patch-package
--- a/node_modules/lodash/flip.js
+++ b/node_modules/lodash/flip.js
@@ -25,4 +25,4 @@ function flip(func) {
   return createWrap(func, WRAP_FLIP_FLAG);
 }
 
-module.exports = flip;
+module.patchPackage = flip;
deleted file mode 100644
--- a/node_modules/lodash/fp/__.js
+++ /dev/null
@@ -1 +0,0 @@
-module.exports = require('./placeholder');
new file mode 100644
--- /dev/null
+++ b/node_modules/lodash/newFile.md
@@ -0,0 +1 @@
+this is a new file
END SNAPSHOT"
`;

exports[`Test include-exclude-paths: only __.js being deleted 1`] = `
"SNAPSHOT: only __.js being deleted
patch-package
deleted file mode 100644
--- a/node_modules/lodash/fp/__.js
+++ /dev/null
@@ -1 +0,0 @@
-module.exports = require('./placeholder');
END SNAPSHOT"
`;

exports[`Test include-exclude-paths: only base files, no clone files 1`] = `
"SNAPSHOT: only base files, no clone files
patch-package
--- a/node_modules/lodash/_baseClamp.js
+++ b/node_modules/lodash/_baseClamp.js
@@ -19,4 +19,4 @@ function baseClamp(number, lower, upper) {
   return number;
 }
 
-module.exports = baseClamp;
+module.patchPackage = baseClamp;
END SNAPSHOT"
`;
